src/backend/logconfig.py

```diff
@@ -22,26 +22,36 @@ def filter(self, record: logging.LogRecord) -> bool:
         return not record.name.startswith("watchfiles")
 
 
-fileLoggingHandler = logging.handlers.RotatingFileHandler(
-    filename=os.path.join(
-        LOGGING_PATH,
-        f"gitalytics.log"
-    ),
-    maxBytes=1024 * 1024 * 10,  # roughly 10mb
-    backupCount=5,
+debugHandler = logging.handlers.TimedRotatingFileHandler(
+    filename=os.path.join(LOGGING_PATH, "backend-debug.log"),
```

Comment:
```suggestion
    filename=os.path.join(LOGGING_PATH, "backend-info.log"),
```
Auf production wäre es ja immer info.